andi/values.yaml

```diff
@@ -0,0 +1,36 @@
+replicaCount: 1
+
+image:
+  repository: smartcitiesdata/andi
+  tag: 0.2.3
+  pullPolicy: Always
+
+monitoring:
+  targetPort: 80
+
+redis:
+  host: redis.external-services
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: NodePort
+  port: 80
+  targetPort: 80
+
+ingress:
+  enabled: false
+  scheme: ""
+  subnets: ""
+  securityGroups: ""
+  certificateARN: ""
+  dnsZone: "localhost"
+
+resources:
```

Comment:
This hasn't been followed as strictly, but we should really only have these resources in our overrides (deploy)  files, and just let the defaults be defaults in our charts-default.

Reply:
Can you say defaults a few more times?

But seriously, what are you trying to convey?